docs/en/modules/apa102.md

```diff
@@ -1,7 +1,7 @@
 # APA102 Module
 | Since  | Origin / Contributor  | Maintainer  | Source  |
 | :----- | :-------------------- | :---------- | :------ |
-| 2016-01-26 | [Robert Foss](https://github.com/robertfoss)| [Robert Foss](https://github.com/robertfoss)| [apa102.c](../../../app/modules/apa102.c)|
+| 2016-01-26 | [Robert Foss](https://github.com/robertfoss)| [Robert Foss](https://github.com/robertfoss)| [Felix Penzlin](https://github.com/FelixPe)| [apa102.c](../../../app/modules/apa102.c)|
```

Comment:
What was the _intended_ change here? You effectively removed the source code reference.

IMHO this change should be reverted completely because a) "Origin / Contributor" doesn't change once a module has been contributed and b) changing the maintainer doesn't just happen because you add your name there.